packages/action-menu/src/ActionMenu.ts

```diff
@@ -65,7 +65,7 @@ export class ActionMenu extends ObserveSlotText(PickerBase, 'label') {
                 ?quiet=${this.quiet}
                 ?selected=${this.open}
                 aria-haspopup="true"
-                aria-controls="popover"
+                aria-controls=${ifDefined(this.open ? 'menu' : undefined)}
```

Comment:
We should probably add this same thing [here](https://github.com/adobe/spectrum-web-components/blob/main/packages/split-button/src/SplitButton.ts#L152)?